tests/test_core.py

```diff
@@ -241,20 +242,36 @@ def test_region_processing_weighted_aggregation(folder, exp_df, args, caplog):
         assert logmsg in caplog.text
 
 
-def test_region_processing_skip_aggregation():
-    test_df = IamDataFrame(
+@pytest.mark.parametrize(
+    "model_name, region_names",
+    [("m_a", ("region_A", "region_B")), ("m_b", ("region_A", "region_b"))],
```

Comment:
For consistency and easier readability
```suggestion
    [("model_a", ("region_A", "region_B")), ("model_b", ("region_A", "region_b"))],
```